src/main/kotlin/no/nav/syfo/behandlendeenhet/api/BehandlendeEnhetDTO.kt

```diff
@@ -1,6 +1,6 @@
 package no.nav.syfo.behandlendeenhet.api
 
-data class PersonDTO(
+data class BehandlendeEnhetDTO(
     val personident: String,
     val isNavUtland: Boolean,
     val oppfolgingsenhet: String?,
```

Comment:
Er tanken etterhvert at `isNavUtland` skal fases ut slik at `oppfolgingsenhet` kan holde på dette? Nav utland blir vel en form for oppfølgingsenhet, så `isNavUtland` blir kanskje redundant? 🤔

Reply:
Ja, tanken er at dette skal tas bort når det ikke brukes lengre fra `syfomodiaperson`.